studiopress/genesis-custom-blocks
# Add template editor rendering

#### Changes
- [x] Renders the CSS in the React editor
- [x] Renders the markup in the React template editor on the front-end and the 'Front-end Preview':
![the-markup9on](https://user-images.githubusercontent.com/4063887/116318936-3c55bd00-a77b-11eb-9f0d-5b11ee7171d1.gif)


#### Testing instructions
1. `composer install && npm install && npm run dev`
2. `/wp-admin` > Custom Blocks > Add New
3. Give the block a title of 'Testing Template'
4. Add a field
5. Give the field a label of 'Text': 
![the-field-label](https://user-images.githubusercontent.com/4063887/117112449-adced600-ad4e-11eb-97e2-73315937070e.png)
6. Click 'Template Editor'
7. Enter this:
```html
<div class="foo">
    {{text}}
</div>
``` 
![templa-sdf](https://user-images.githubusercontent.com/4063887/117113228-cd1a3300-ad4f-11eb-9f9c-e2c660dc038d.png)

8. Click the 'CSS' button: 
![the-sdf](https://user-images.githubusercontent.com/4063887/117112658-03a37e00-ad4f-11eb-9937-d1fd11383eab.png)
9. Enter this:
```css
.foo {
    color: green;
}
```
10. Click 'Save draft'
11. Click 'Editor Preview
12. Enter some text: 
![entering-in](https://user-images.githubusercontent.com/4063887/117112864-4ebd9100-ad4f-11eb-9633-72448aad8f37.png)
13. Click 'Front-end Preview'
14. Expected: The text you entered is there, and it's green: 
![when-ijsd](https://user-images.githubusercontent.com/4063887/117112963-74e33100-ad4f-11eb-8e42-2689cd251465.png)
